p2p/host/autonat/autonat_test.go

```diff
@@ -121,8 +122,8 @@ func TestAutoNATPrivate(t *testing.T) {
 	connect(t, hs, hc)
 	require.Eventually(t,
 		func() bool { return an.Status() == network.ReachabilityPrivate },
-		2*time.Second,
-		25*time.Millisecond,
+		testutils.ScaleDuration(2*time.Second),
```

Comment:
Any idea why this takes so long? I would've expected 2s to be a long timeout already.

Reply:
I think it's this second: https://github.com/libp2p/go-libp2p/blob/e732bda3fa076707e2837eb9edfe8955ee8f4ad8/p2p/host/autonat/autonat.go#L256-L262. The first timer fires immediately, but doesn't have any peer ID yet. The second timer fires after one second.

Seems like 20s is way too much on CI. Maybe we can just set to 3s?

Reply:
It's the upper limit, but sure updated :)